backend/src/packages/business/business.controller.ts

```diff
@@ -352,11 +354,13 @@ class BusinessController extends Controller {
       authStrategy: defaultStrategies,
       validation: {
         params: driverGetParameters,
```

Comment:
we don't need to pass businessId as param
/business/drivers, and then by authorized user Id on be find businessId on service layer

Reply:
Should I rewrite add driver without businessId?